tests/test_gmsh_IO.py

```diff
@@ -318,3 +318,89 @@ def test_generate_mesh(self):
 
         # check if the coordinates of the points are correct
         TestUtils.assert_dictionary_almost_equal(expected_mesh_data, mesh_data)
+
+    def test_geometry_data_2D(self):
```

Comment:
In test test, the only thing you check if geometry data has physical groups. So can you you rename this test accordingly. Also correct the docstring